src/commands/node/start.ts

```diff
@@ -36,7 +37,7 @@ export class StartNode extends SwankyCommand<typeof StartNode> {
     // Non-Persistent mode (`--dev`) allows all CORS origin, without `--dev`, users need to specify origins by `--rpc-cors`.
     await execaCommand(
       `${this.swankyConfig.node.localPath} \
-      ${this.swankyConfig.node.version === "1.6.0" ? `--finalize-delay-sec ${flags.finalizeDelaySec}` : ""} \
+      ${ semver.gt(this.swankyConfig.node.version, "1.6.0") ? `--finalize-delay-sec ${flags.finalizeDelaySec}` : ""} \
```

Comment:
`semver.gte` instead since versions higher or equal to "1.6.0" are valid.
I am pushing a commit since it is the only fix remaining